### PR TITLE
Add Fractal Rate Oracle Addresses + Proofing Contracts

### PR DESCRIPTION
**Pull request by tom2o17**

### CODE REVIEW

File: src/Constants.sol [hunk 1]
```diff
@@ -210,7 +210,7 @@ abstract contract Helper is TestBase {
         vm.label(0x32467a5fc2d72D21E8DCe990906547A2b012f382, "Constants.WBTC_FRAX_PAIR_ADDRESS");
         vm.label(0x794F6B13FBd7EB7ef10d1ED205c9a416910207Ff, "Constants.WETH_FRAX_PAIR_ADDRESS");
         vm.label(0xc9268c7F59850B26567b0f5d3Dcc317cEDC43490, "Constants.SLIPPAGE_AUCTION_FACTORY");
-        vm.label(0xa8478781D9605E17c3b4c9C79e7127c4597116E4, "Constants.FXB_FACTORY");
+        vm.label(0x7a07D606c87b7251c2953A30Fa445d8c5F856C7A, "Constants.FXB_FACTORY");
```
Comment (tom2o17) [inline src/Constants.sol:213]: cc @pegahcarter 

Curious which is current

Comment (pegahcarter) [reply src/Constants.sol:213]: The change you made here to `0x7a07` is proper.

Confirmation [link](https://etherscan.io/address/0x452420df4AC1e3db5429b5FD629f3047482C543C#readContract#F11).

File: src/Constants.sol [hunk 2]
```diff
@@ -314,6 +314,18 @@ library FraxtalL2 {
     address internal constant STATE_ROOT_ORACLE = 0xeD403d48e2bC946438B5686AA1AD65056Ccf9512;
     address internal constant MERKLE_PRICE_SOURCE = 0x9990eb28E58380122AA3AdF64ea6f874df51114C;
     address internal constant CHAINLINK_MERKLE_PRICE_SOURCE = 0x4AD728706bFe28A3F090b3f1D8Ac3A8515805B41;
+    address internal constant SFRXETH_L2_RATE_ORACLE = 0xEE095b7d9191603126Da584a1179BB403a027c3A;
+    address internal constant SFRAX_L2_RATE_ORACLE = 0x0D7888e8aC9e5D0915148E0F871a3893aF3d2aaF;
+    address internal constant SDAI_L2_RATE_ORACLE = 0xfdE8C36F32Bf32e73A1bdeb4ef3E17709674a838;
+    address internal constant SUSDE_L2_RATE_ORACLE = 0xd295936C8Bb465ADd1eC756a51698127CB4F4910;
+    address internal constant SFRXETH_L2_RATE_ORACLE_OLD = 0x8865435777730eAAbAAF2d1F55F115a87AbCf91A;
```
Comment (pegahcarter) [inline src/Constants.sol:321]: This should be fine assuming there's no real need to actively check the old oracle as a second update would overrwrite the old oracle and require looking at the diff history.

Comment (tom2o17) [reply src/Constants.sol:321]: Good catch there is no need to update/point to `0x0D7888e8aC9e5D0915148E0F871a3893aF3d2aaF`

Comment (pegahcarter) [reply src/Constants.sol:321]: I'm confused here as I was referring to the use of `_OLD` variables, although if there's a fix from this comment that works too.

Comment (tom2o17) [reply src/Constants.sol:321]: @pegahcarter Yeah so the reason we have the old Oracles is becuase they are still being used by the curve pools on fraxtal and are declared as immutable. 

As a consequence, we will still support them going forward. Unless we want to do a liquidity migration on these pools. But none of the changes from the audit impact the manner in which these oracles are utilized by the pools.

**Pull request merged**